p2p/stream/common/streammanager/config.go

```diff
@@ -0,0 +1,25 @@
+package streammanager
+
+import "time"
+
+const (
+	// checkInterval is the default interval for checking stream number. If the stream
+	// number is smaller than softLoCap, an active discover through DHT will be triggered.
+	checkInterval = 30 * time.Second
```

Comment:
how are these numbers chosen?

Reply:
By estimation without solid evidence. Basically this is just like estimating the timeout to do a HTTP request. The only trick here is to make discTimeout less than checkInterval to avoid simultaneous host/client stream setup. Do you have other references that these numbers can be analyzed upon?

Reply:
We can probably observe the timeout by adding prometheus metrics and change to a more solid value after launch. For now I think the number is good to start with. Do you other references that can be helped to determine these number's values?